src/main/java/com/OptimisticChemicalMakers/MapFood/controllers/ProductController.java

```diff
@@ -24,8 +20,8 @@
 	@Autowired
 	private ProductService productService;
 	
-	@GetMapping(value = "/productByStore/{restaurantId}")
-    public List<ProductDto> getProductByStore(@PathVariable String restaurantId) {
-		return productService.getByRestaurantId(restaurantId);
+	@GetMapping(value = "/productByStore/{hashRestaurant}")

```

Comment:
Pelo que sei para formar as URIs da aplicação o ideal é não utilizar cammel case. Na separação de palavras talvez o ideal é usar hífen. Esses dias ainda estava lendo sobre isso e encontrei [esse](https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris) post no  Stack Overflow.